frontend/web/src/styles.css

```
@@ -1,118 +1,173 @@
-  --color-error: oklch(57% 0.215 22);
-  --color-error-content: oklch(99% 0 0);
-  --radius-selector: 0.5rem;
-  --radius-field: 0.25rem;
-  --radius-box: 0.5rem;
-  --size-selector: 0.25rem;
-  --size-field: 0.25rem;
-  --border: 1px;
-  --depth: 1;
-  --noise: 0;
-}
+@custom-variant dark (&:is(.dark *));
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)